### PR TITLE
Add GetServiceCapabilities for media ver10

### PR DESCRIPTION
**Pull request by waranlogesh**

### CODE REVIEW

File: test/media/get_service_capabilities_test.exs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+defmodule Onvif.Media.Ver10.GetServiceCapabilitiesTest do
+  use ExUnit.Case, async: true
+
+  @moduletag capture_log: true
+
+  describe "GetServiceCapabilities/1" do
+    test "should parse with correct dvalues and defaults for non existing attributes" do
```
Comment (waranlogesh) [inline test/media/get_service_capabilities_test.exs:7]:
```suggestion
    test "should parse with correct values and defaults for non existing attributes" do
```

File: lib/media/ver10/service_capabilities.ex [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  ]
+  embedded_schema do
+    field(:snapshot_uri, :boolean, default: false)
+    field(:rotation, :boolean, default: false)
+    field(:video_source_mode, :boolean, default: false)
+    field(:osd, :boolean, default: false)
+    field(:temporary_osd_text, :boolean, default: false)
+    field(:exi_compression, :boolean, default: false)
+
+    embeds_one :profile_capabilities, ProfileCapailities, primary_key: false, on_replace: :update do
```
Comment (hammeraj) [inline lib/media/ver10/service_capabilities.ex:27]:
```suggestion
    embeds_one :profile_capabilities, ProfileCapabilities, primary_key: false, on_replace: :update do
```

File: test/media/get_service_capabilities_test.exs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      Mimic.expect(Tesla, :request, fn _client, _opts ->
+        {:ok, %{status: 200, body: xml_response}}
+      end)
+
+      {:ok, service_capabilities} = Onvif.Media.Ver10.GetServiceCapabilities.request(device)
+
+      assert service_capabilities == %Onvif.Media.Ver10.ServiceCapabilities{
+               exi_compression: false,
+               osd: true,
+               profile_capabilities: %Onvif.Media.Ver10.ServiceCapabilities.ProfileCapailities{
```
Comment (hammeraj) [inline test/media/get_service_capabilities_test.exs:21]:
```suggestion
               profile_capabilities: %Onvif.Media.Ver10.ServiceCapabilities.ProfileCapabilities{
```

**Review by hammeraj — APPROVED** [review]
Tested with Homaxi standalone and Uniview NVR

**Pull request merged**